evennia/web/webclient/static/webclient/js/webclient_gui.js

```diff
@@ -12,618 +12,258 @@
  *
  */
```

Comment:
This docstring needs to be updated to indicate that this is now basically a plugin-handler. It also needs a summary of how a plugin must look to work and how to add it. Finally now that this is made to be more replaceable by users it might be a good time to make a listing of the supported Events and what each event expects from a plugin that uses it.